apps/i18n/tutorialExplorer/en_us.json

```diff
@@ -0,0 +1,46 @@
+{
+  "filterGrades": "Grades",
+  "filterGradesPre": "Pre-reader",
+  "filterGrades25": "Grades 2-5",
+  "filterGrades68": "Grades 6-8",
+  "filterGrades9": "Grades 9+",
+  "filterTeacherExperience": "Educator's experience",
```

Comment:
Educator experience

I listed these text changes as P3 for later, but may be easier to just update here already?